src/transbank_webpay.php

```diff
@@ -70,6 +74,8 @@ function __construct(&$subject, $config) {
                 $this->createPdf();
             } else if (isset($_GET['updateConfig'])) {
                 $this->updateConfig();
+                $this->sendPluginVersion();
```

Comment:
Estas lineas me llaman la atención, no se si son necesarias. Sobre todo por el print_r.
```suggestion
                $this->sendPluginVersion();
```